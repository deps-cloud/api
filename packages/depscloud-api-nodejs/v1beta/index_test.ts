describe("v1beta", () => {
    test("require", () => {
        const schema = require("./index");

        expect(Object.keys(schema)).toMatchSnapshot();
    });
});
