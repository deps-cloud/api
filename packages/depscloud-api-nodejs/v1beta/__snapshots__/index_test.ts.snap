// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`v1beta require 1`] = `
Array [
  "ProviderURL",
  "Source",
  "Module",
  "SourceModule",
  "ModuleDependency",
  "ManifestDependency",
  "ManifestFile",
  "MatchRequest",
  "MatchResponse",
  "ExtractRequest",
  "ExtractResponse",
  "ManifestExtractionService",
  "StoreRequest",
  "StoreResponse",
  "ManifestStorageService",
  "ListRequest",
  "ListSourcesResponse",
  "ListModulesResponse",
  "ManagedModule",
  "ManagedSource",
  "ListManagedSourcesResponse",
  "ListManagedModulesResponse",
  "SourceService",
  "ModuleService",
  "Dependency",
  "DependentsResponse",
  "DependenciesResponse",
  "SearchRequest",
  "SearchResponse",
  "TraversalService",
]
`;
